airflow/dags/ingestion/ask-astro-load.py

```diff
@@ -200,66 +217,68 @@ def extract_astro_blogs():
 
         return [df]
 
-    _check_seed_baseline = check_seed_baseline()
-
     md_docs = extract_github_markdown.expand(source=markdown_docs_sources)
-
-    rst_docs = extract_github_rst.expand(source=rst_docs_sources)
-
-    issues_docs = extract_github_issues.expand(repo_base=issues_docs_sources)
-
-    stackoverflow_docs = extract_stack_overflow.partial(stackoverflow_cutoff_date=stackoverflow_cutoff_date).expand(
-        tag=stackoverflow_tags
-    )
-
+    issues_docs = extract_github_issues.expand(source=issues_docs_sources)
+    stackoverflow_docs = extract_stack_overflow.expand(tag=stackoverflow_tags)
     # slack_docs = extract_slack_archive.expand(source=slack_channel_sources)
-
     registry_cells_docs = extract_astro_registry_cell_types()
-
     blogs_docs = extract_astro_blogs()
-
     registry_dags_docs = extract_astro_registry_dags()
+    code_samples = extract_github_python.expand(source=code_samples_sources)
+    _airflow_docs = extract_airflow_docs()
 
-    code_samples = extract_github_python.partial().expand(source=code_samples_sources)
+    _get_schema = get_schema(schema_file="include/data/schema.json")
+    _check_schema = check_schema(class_objects=_get_schema)
+    _create_schema = create_schema(class_objects=_get_schema)
+    _check_seed_baseline = check_seed_baseline(seed_baseline_url=seed_baseline_url)
 
     markdown_tasks = [
         md_docs,
-        rst_docs,
         issues_docs,
         stackoverflow_docs,
         # slack_docs,
         blogs_docs,
         registry_cells_docs,
     ]
 
+    html_tasks = [_airflow_docs]
+
     python_code_tasks = [registry_dags_docs, code_samples]
 
     split_md_docs = task(split.split_markdown).expand(dfs=markdown_tasks)
 
     split_code_docs = task(split.split_python).expand(dfs=python_code_tasks)
 
-    task.weaviate_import(ingest.import_data, weaviate_conn_id=_WEAVIATE_CONN_ID, retries=10, retry_delay=30).partial(
-        class_name=WEAVIATE_CLASS
-    ).expand(dfs=[split_md_docs, split_code_docs])
+    split_html_docs = task(split.split_html).expand(dfs=html_tasks)
+
+    _import_data = (
+        task(ask_astro_weaviate_hook.ingest_data, retries=10)
+        .partial(
+            class_name=WEAVIATE_CLASS,
+            existing="upsert",
+            doc_key="docLink",
+            batch_params={"batch_size": 1000},
+            verbose=True,
+        )
+        .expand(dfs=[split_md_docs, split_code_docs, split_html_docs])
+    )
 
-    _import_baseline = task.weaviate_import(
-        ingest.import_baseline, trigger_rule="none_failed", weaviate_conn_id=_WEAVIATE_CONN_ID
-    )(class_name=WEAVIATE_CLASS, seed_baseline_url=seed_baseline_url)
+    _import_baseline = task(ask_astro_weaviate_hook.import_baseline, trigger_rule="none_failed")(
```

Comment:
Should call `include.tasks.ingest.import_baseline()` which then calls the hook function.  This way we can keep astro specific logic in the DAG/task code.

Reply:
Isn't it still the same thing? We would be calling the hook at a separate file in a different method. I think this can be changed during Phase 2.5 when we completely use the Weaviate hook

Reply:
This code is specific to ask-astro. Is there a reason we should change this? For LLM providers we will anyways create different hook looking at this implementation